CHANGELOG.md

```diff
@@ -12,12 +12,26 @@ accidentally triggering the load of a previous DB version.**
 * #5246 Make connection establishment interruptible
 
 **Bugfixes**
-* #4804 Skip bucketing when start or end of refresh job is null
 * #4926 Fix corruption when inserting into compressed chunks
 * #5218 Add role-level security to job error log
 * #5214 Fix use of prepared statement in async module
-* #5259 Lock down search_path in SPI calls
+
+## 2.9.3 (2023-02-04)
```

Comment:
I am pretty sure that this will not go out on Saturday (2023-02-04) 

Setting to today and update if we don't get it on time. 

```suggestion
## 2.9.3 (2023-02-03)
```